src/views/tickets/orders-rsvp.php

```diff
@@ -44,12 +44,16 @@
 					esc_html__( ' on %s', 'event-tickets' ),
 					date_i18n( 'F j, Y', strtotime( esc_attr( $first_attendee['purchase_time'] ) ) )
 				);
+
+				/**
+				* Inject content into the RSVP User Details block on the orders page
+				*
+				* @param array $attendee_group Attendee array
+				* @param WP_Post $post_id Post object that the tickets are tied to
+				*/
+				do_action( 'event_tickets_user_details_rsvp', $attendee_group, $post_id );
```

Comment:
I feel like this action should occur outside of the `<p/>` tag unless you strongly disagree :)